runtime/doc/astro.txt

```diff
@@ -0,0 +1,24 @@
+*astro.txt* Astro language indentation and syntax support in Vim or Neovim.
```

Comment:
is this for the filetype or syntax? Depending on the answer, this belongs into doc/filetype.txt or doc/syntax.txt